packages/docusaurus-theme-classic/src/theme/SiteMetadata/index.tsx

```diff
@@ -33,16 +33,22 @@ function AlternateLangHeaders(): JSX.Element {
   return (
     <Head>
       {Object.entries(localeConfigs).map(([locale, {htmlLang}]) => (
-        <link
-          key={locale}
-          rel="alternate"
-          href={alternatePageUtils.createUrl({
-            locale,
-            fullyQualified: true,
-          })}
-          hrefLang={htmlLang}
-        />
+		    <>
```

Comment:
Yes 👍 

@NotHr I think you could loop twice instead of using a fragment. 

Probably not a big deal but it would look cleaner if `hrefLang` and `og:locale:alternate` were not "interleaved" in the final html markup.